### DISCUSSION TITLE
How to set target for backup  on a NVMe drive

### DISCUSSION THREAD

**Discussion by phil-Xbones** [outdoorbits/little-backup-box#311] [General]
I made a " copy " PioB-1999's   LBB and can not configure my NVMe as target for backup .

 I can see it with ssh lspci 

outdoorbits asked for  the answer of the following command:
sudo python3 /var/www/little-backup-box/lib_storage.py --Action get_available_partitions --skipMounted False --ignore-fs True

/dev/sda: /dev/sda
/dev/sda1: --uuid 10CE6071CE605150
/dev/nvme0n1: /dev/nvme0n1

thanks for your help

phil

**Comment by outdoorbits** [comment]
O.k., NVME is there. Is it formatted? What says
`sudo python3 /var/www/little-backup-box/lib_storage.py --Action get_available_partitions --skipMounted False --ignore-fs False`
To be clear: You don't see buttons for NVME in the backup page?
In Tools, con you select the NVME for formatting?

**Reply by phil-Xbones** [reply]
Hello   
very good catch, i thought the NVMe crucial 1T was already formated : works good now .

do you have a tips  for extracting the pictures from the NVMe to a windows computer once back to home without dismounting the LBB and using a NVMe to usb adapter ?

thanks a lot 

phil

**Comment by outdoorbits** [comment]
> you can Message ID: 
> ***@***.***>

You can access lbb via samba file share. (Explorer - Network)

**Reply by phil-Xbones** [reply]
thanks  works  fine with filezilla and a RJ 45 cable, much faster than wifi 

sorry to ask stupid noobs question ;-)
